server/templates/pages/src/index.html

```diff
@@ -33,6 +33,8 @@
           <script src="/bower_components/html5shiv/dist/html5shiv.js"></script>
           <!-- endbuild -->
         <![endif]-->
+
+        <meta name="fxa-content-server/config" content="{{ config }}" />
```

Comment:
should we put this with the rest of the `meta` elements before other css / js stuff?